CHANGES.md

```diff
@@ -19,6 +19,7 @@ We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (
 * Bump default `shfmt` version to latest `3.7.0` -> `3.8.0`. ([#2050](https://github.com/diffplug/spotless/pull/2050))
 * Bump default `ktlint` version to latest `1.1.1` -> `1.2.1`. ([#2057](https://github.com/diffplug/spotless/pull/2057))
 * Bump default `sortpom` version to latest `3.4.0` -> `3.4.1`. ([#2078](https://github.com/diffplug/spotless/pull/2078))
+* Default `ktlint` editorconfig `ktlint_code_style` to `intellij_idea` -> `ktlint_official`. ([#2090](https://github.com/diffplug/spotless/pull/2090))
```

Comment:
Actually, the context was https://github.com/diffplug/spotless/pull/1808#discussion_r1329008496.

Reply:
Maybe we can just note the default style `intellij_idea` in ktlint sections in docs, users can override them anyway in their configs.

Reply:
Yes, developer can override the style.
But the problem is that I introduced.
> According Ktlint documentation, only one want to revert another code style must set the ktlint_code_style, but in Spotless need to set the ktlint_code_style who want to apply ktlint_official or else styles.

Reply:
That's the ktlint CLI's default value instead of this plugin.

Did you see the context before? This change will break users' compatibility, it's so painful and annoying.

Reply:
No, I don't think that break users' compatibility.
This ```ktlint_official``` is ktlint's default value, and spotless loads ktlint plugin, so that there's no reason doesn't follow ktlint official documentation.

See below snippet 
```gradle
spotless {
  kotlin {
    ktlint()
  }
}
```

You say that spotless will apply ```intellij_idea``` style ktlint.
But the one who read ktlint documentation, will say that spotless will apply ```ktlint_official``` style ktlint.
This inconsistent situation will bring more fatigue to developer. Every developers must double check both spotless documentation and ktlint documentation what is different.
It is non-sense.